plugins/jobs/src/js/JobsMenu.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import { componentFromStream } from "data-service";
+import "rxjs/add/operator/combineLatest";
+
+import Page from "#SRC/js/components/Page";
+import JobsBreadcrumbs from "#SRC/js/components/breadcrumbs/JobsBreadcrumbs";
+import { jobsRunNowAction, jobsRunNow$ } from "./JobsRunNow";
+
+const getActions = function(props) {
+  const [edit, ...rest] = props.actions;
+  const newActions = [];
+
+  newActions.push(edit);
+
+  newActions.push(jobsRunNowAction(props.job.getId()));
+
+  return newActions.concat(rest);
+};
+
+const JobsMenu = componentFromStream(props$ => {
+  return props$.combineLatest(jobsRunNow$.startWith({}), props => {
```

Comment:
Why do we need `jobRunNow$`? I can't see that we are utilizing any of its observables output

Reply:
When I remove this and just do ```props$.subscribe(props => {})``` I get ```cannot use props$ undefined```. I guess we do this to extend with an observable that always exist. Maybe @Fabs can clear this out a bit.

Reply:
`combineLatest` will not emit unless both the streams it is combining emits at least once. So if you combinLatest the props with jobsRunNow, the menu will never appear. You need to give jobsRunNow an initial value.

Reply:
I get that, but `jobRunNow$` is called with `.next({id})` and this id property is not used here as far as I can see, so why do we need the value of the stream?

Reply:
You need to pass the id to the Subject, to trigger the runNow of the particular id. 

jobRunNow$ is a stream of all the results of the executions of the graphqlMutation to run a job. To issue the command you have to onNext to the Subject related to that, so the subject will switchMap into the request.